src/system/RoutesModule/Controller/RouteController.php

```diff
@@ -49,56 +63,140 @@ public function indexAction(Request $request)
+     *        defaults = {"sort" = "", "sortdir" = "asc", "pos" = 1, "num" = 0, "_format" = "html"},
+     *        methods = {"GET"}
+     * )
+     *
+     * @param Request $request Current request instance.
+     * @param string  $sort    Sorting field.
+     * @param string  $sortdir Sorting direction.
+     * @param int     $pos     Current pager position.
+     * @param int     $num     Amount of entries to display.
+     *
+     * @return mixed Output.
+     *
+     * @throws AccessDeniedException Thrown if the user doesn't have required permissions.
+     */
+    public function adminViewAction(Request $request, $sort, $sortdir, $pos, $num)
+    {
+        return parent::adminViewAction($request, $sort, $sortdir, $pos, $num);
+    }
+
+    /**
+     * This action provides an item list overview.
      *
      * @Route("/routes/view/{sort}/{sortdir}/{pos}/{num}.{_format}",
      *        requirements = {"sortdir" = "asc|desc|ASC|DESC", "pos" = "\d+", "num" = "\d+", "_format" = "html|kml"},
      *        defaults = {"sort" = "", "sortdir" = "asc", "pos" = 1, "num" = 0, "_format" = "html"},
      *        methods = {"GET"}
      * )
      *
-     * @param Request  $request      Current request instance
-     * @param string  $sort         Sorting field.
-     * @param string  $sortdir      Sorting direction.
-     * @param int     $pos          Current pager position.
-     * @param int     $num          Amount of entries to display.
-     * @param string  $tpl          Name of alternative template (to be used instead of the default template).
+     * @param Request $request Current request instance.
+     * @param string  $sort    Sorting field.
+     * @param string  $sortdir Sorting direction.
+     * @param int     $pos     Current pager position.
+     * @param int     $num     Amount of entries to display.
      *
      * @return mixed Output.
      *
      * @throws AccessDeniedException Thrown if the user doesn't have required permissions.
      */
     public function viewAction(Request $request, $sort, $sortdir, $pos, $num)
     {
+        return parent::viewAction($request, $sort, $sortdir, $pos, $num);
+    }
+
+    /**
+     * This method includes the common implementation code for adminView() and view().
+     */
+    protected function viewInternal(Request $request, $sort, $sortdir, $pos, $num, $isAdmin = false)
+    {
         // Always force to see all entries to make sortable working.
         $request->query->set('all', 1);
 
-        $groupMessages = [
-            RouteEntity::POSITION_FIXED_TOP => $this->__('Routes fixed to the top of the list:'),
-            RouteEntity::POSITION_MIDDLE => $this->__('Normal routes:'),
-            RouteEntity::POSITION_FIXED_BOTTOM => $this->__('Routes fixed to the bottom of the list:'),
-        ];
-        $this->view->assign('groupMessages', $groupMessages);
-        $this->view->assign('sortableGroups', [RouteEntity::POSITION_MIDDLE]);
```

Comment:
No, it is not gone, but moved to https://github.com/zikula/core/blob/1.4/src/system/RoutesModule/Helper/ViewHelper.php#L33